### PR TITLE
[Fillter] Change custom-easy filter version from v0 to v1 

### PR DESCRIPTION
**Pull request by gichan-jang**
Change custom-easy filter version from v0 to v1

 *todo: Support dynamic invoke of custom-easy filter

**Self evaluation:**
1. Build test: [* ]Passed [ ]Failed [ ]Skipped
2. Run test: [* ]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: gst/nnstreamer/tensor_filter/tensor_filter_custom_easy.c [hunk 1]
```diff
@@ -171,75 +173,101 @@ custom_open (const GstTensorFilterProperties * prop, void **private_data)
 /**
  * @brief Callback required by tensor_filter subplugin
  */
-static int
-custom_invoke (const GstTensorFilterProperties * prop,
-    void **private_data, const GstTensorMemory * input,
-    GstTensorMemory * output)
+static void
```
Comment (songgot) [inline gst/nnstreamer/tensor_filter/tensor_filter_custom_easy.c:176]: The return type is 'void', did you intend?

File: gst/nnstreamer/tensor_filter/tensor_filter_custom_easy.c [hunk 2]
```diff
@@ -171,75 +173,101 @@ custom_open (const GstTensorFilterProperties * prop, void **private_data)
...
+custom_close (const GstTensorFilterProperties * prop, void **private_data)
 {
   runtime_data *rd = *private_data;
+  UNUSED (prop);
+  g_free (rd);
+  *private_data = NULL;
+}
+
+
```
Comment (songgot) [inline gst/nnstreamer/tensor_filter/tensor_filter_custom_easy.c:185]: let's remove this line.

**Review by songgot — APPROVED** [review]
LGTM

File: gst/nnstreamer/tensor_filter/tensor_filter_custom_easy.c (hunk 1 above)
Comment (gichan-jang) [reply gst/nnstreamer/tensor_filter/tensor_filter_custom_easy.c:176]: It's defined in `nnstreamer_plugin_api_filter.h`
https://github.com/nnstreamer/nnstreamer/blob/e2494ce89baf53ef8ce80e5272c71512891c89fe/gst/nnstreamer/include/nnstreamer_plugin_api_filter.h#L291

File: gst/nnstreamer/tensor_filter/tensor_filter_custom_easy.c (hunk 2 above)
Comment (gichan-jang) [reply gst/nnstreamer/tensor_filter/tensor_filter_custom_easy.c:185]: Thanks. Removed.

File: gst/nnstreamer/include/nnstreamer_plugin_api_filter.h [hunk 3]
```diff
@@ -183,14 +184,15 @@ typedef struct _GstTensorFilterFrameworkInfo
 {
   const char *name; /**< Name of the neural network framework, searchable by FRAMEWORK property. Subplugin is supposed to allocate/deallocate. */
   int allow_in_place; /**< TRUE(nonzero) if InPlace transfer of input-to-output is allowed. Not supported in main, yet */
-  int allocate_in_invoke; /**< TRUE(nonzero) if invoke_NN is going to allocate outputptr by itself and return the address via outputptr. Do not change this value after cap negotiation is complete (or the stream has been started). */
+  int allocate_in_invoke; /**< TRUE(nonzero) if invoke_NN is going to allocate outputptr by itself and return the address via outputptr. Do not change this value after cap negotiation is complete (or the stream has been started). In the case of dynamic invoke, allocate_in_invoke should be TRUE.*/
```
Comment (myungjoo) [inline gst/nnstreamer/include/nnstreamer_plugin_api_filter.h:187]: Add a condition-check somewhere to enforce the rule, ```In the case of dynamic invoke, allocate_in_invoke should be TRUE.```.

LGTM other than this.
@jaeyun-jung or @again4you : please take a look and proceed.

**Comment by jaeyun-jung** [comment]
need rebase

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**